### ISSUE TITLE
Combat Tracker / Party editing causes logout

### ISSUE THREAD

**Issue by BillyKings173** [Orcpub/orcpub#338]
**Describe the bug**
After putting character sheets created by another user into a party, using the combat tracker or browsing the sheet in the party tab causes a logout of DMV. Any time a change is made to the sheet, such as casting a spell, a logout is forced.

**To Reproduce**

--First have a user share a sheet via web link--
1. When viewing character, select add to party
2. Create a party or add the character to an existing party

--Then---
1. View/Edit in the 'Parties' tab under 'Characters'
2. Select the character in your party
3. Edit a field on the character (change armor, weapon, cast a spell)
4. In a few moments, get logged out forcibly

--Or--
1. Go to 'Combat tracker' under 'Encounters' tab
2. Add a party to the combat
3. Select the character in your party
4. Edit a field on the character (change armor, weapon, cast a spell)
5. In a few moments, get logged out forcibly

**Expected behavior**
Expected to run a combat and monitor spells, health, armor, weapons, and features.

**Desktop (please complete the following information):**
 - OS: Windows 10 64-bit
 - Browser: Chrome
 - Version 2.5.0.9 (Build 1/16/2020) "Lurkwood" edition

**Additional context**
I'm attempting to run combats with a party of three, sometimes four, characters made by friends. The tool seems great, and every time I make a change it is saved when I log back in. I'd just like to stay logged in! Thanks for your attention!

**Comment by DivertedCircle** [comment]
Can confirm, I've run into this as well.

**Comment by datdamnzotz** [comment]
Yup, Currently you can not edit characters from other accounts because you are not an owner of that character.  You can only edit characters that are under your logged into account.  

So not really a bug, but a feature enhancement to request/grant access to other accounts/characters to edit them.

Characters you own, can be part of the combat tracker and encounters without any problems.
